routes/create-account.js

```diff
@@ -8,39 +8,57 @@ router.get('/', function(req, res, next) {
 });
 
 router.post('/', async function(req, res, next) {
-    // TODO: Validate input.
-    let email = req.body.email;
+    let email = req.body.email;  // Bootstrap form already validates FORM of email.
```

Comment:
No, bootstrap does **client-side** validation. Do a server-side sanity-check validation as well (unless the email verification code already does this?).